build/prepare-sa-test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+kubectl delete namespace namespace-for-testing
```

Comment:
Wait for delete need to be added as delete project/namespace is async procedure
Not problem for CI as we don't have a project in kind created from scratch or this need to be removed

Reply:
done